src/boot/bootctl.c

```diff
@@ -110,6 +110,24 @@ static const char *pick_efi_boot_option_description(void) {
         return arg_efi_boot_option_description ?: "Linux Boot Manager";
 }
 
+static void check_esp_path_duplicates(dev_t *devtptr) {
+        if (check_esp_duplicates(geteuid() != 0, devtptr) <= 1)
+                return;
+
+        printf("%s", ansi_highlight_red());
+        printf("WARNING: found more than one EFI SYSTEM partition !\n");
```

Comment:
please output with log_warning(), this will color the usual way we color.

no space before `!`

please uppercase first char of sentence, i.e. found → Found.

And also say "on the same disk"

Reply:
ok